src/compiler/checker.ts

```diff
@@ -8976,6 +8980,16 @@ namespace ts {
             return symbol && <GenericType>getTypeOfGlobalSymbol(symbol, arity);
         }
 
+        function getGlobalThisSymbol() {

```

Comment:
This doesn't need to be deferred. Around line 722 we have a `builtinGlobals` map - we should immediately make the symbol and add the symbol to that map, so it exists as a normal symbol in the global scope (like `undefined`).